util/codec/codec.go

```diff
@@ -898,6 +898,9 @@ func CutOne(b []byte) (data []byte, remain []byte, err error) {
 	if err != nil {
 		return nil, nil, errors.Trace(err)
 	}
+	if l < 0 || l > len(b) {
```

Comment:
What would meet this change?

Reply:
It is related to a broken test when adding a new system table. https://github.com/pingcap/tidb/pull/27849/files#r730704314 did similar change so we can remove the change here.

Reply:
Removed.